packages/keyboard-shortcuts/src/store/selectors.js

```diff
@@ -20,6 +25,32 @@ export function getShortcutKeyCombination( state, name ) {
 	return state[ name ] ? state[ name ].keyCombination : null;
 }
 
+/**
+ * Returns a string representing the main key combination for a given shortcut name.
+ *
+ * @param {Object} state          Global state.
+ * @param {string} name           Shortcut name.
+ * @param {string} representation Type of reprensentation. (display, raw, ariaLabel )
+ *
+ * @return {string} Shortcut representation.
+ */
+export function getShortcutRepresentation( state, name, representation = 'display' ) {
+	const shortcut = getShortcutKeyCombination( state, name );
+	if ( ! shortcut ) {
+		return null;
+	}
+
+	const formattingMethods = {
+		display: displayShortcut,
+		raw: rawShortcut,
+		ariaLabel: shortcutAriaLabel,
+	};
```

Comment:
Doubt it has much overhead, and I could grant an argument that it benefits from colocation, but this appears to be constant, could be defined at the top of the file.